README.adoc

```diff
@@ -51,6 +51,7 @@ Thank you for contributing to this repo. When creating a pull request, please fo
 
 . For help on style and structure, refer to the https://documentation.suse.com/style/current[Documentation Style Guide].
 
+. If you want to have the documentation reformatted automatically, you can either use `daps-xmlformat` for single files or `daps xmlformat` for whole documents.
```

Comment:
Sounds like a good idea.
* I tried to shorten the sentence.
* `daps xmlformat` actually does both.

```suggestion
. To reformat the documentation automatically, you can use `daps xmlformat`.
For more information, see https://opensuse.github.io/daps/doc/cha.daps.user.output.html#sec.daps.user.output.xmlformat.
```

Reply:
Note that the style guide states both. That's where I took the information from. Not sure if you want to have that updated, too.